service/templates/projects.html

```diff
@@ -267,6 +268,15 @@ <h2> Settings for [[displayName]]</h2>
           </div>
         </div>
 
+        <div id="logging-info">
+          <div class="h3">Default Logger</div>
+          <div style="left:-15px" class="col-lg-4">
+            <select name="repeatSelect" id="repeatSelect" ng-model="settingsFormData.default_logger">
```

Comment:
Since the rest of the page is using material UI components, let's use a <mat-select> here instead of the HTML select